src/math/Frustum.d.ts

```diff
@@ -26,12 +26,12 @@ export class Frustum {
 	planes: Plane[];
 
 	set(
-		p0?: number,
-		p1?: number,
-		p2?: number,
-		p3?: number,
-		p4?: number,
-		p5?: number
+		p0?: Plane,
```

Comment:
BTW: These are all mandatory parameters.

Reply:
You are correct. PR fixed. An alternative could be to make them non-mandatory in JS, but my current ambition does not go beyond fixing the d.ts to match the js.